src/main/resources/assets/paths/darV3.yaml

```diff
@@ -0,0 +1,25 @@
+post:
```

Comment:
Do we need to add a 400 response for the error thrown if the user does not have all of the DAAs on their LC for any of the datasets in this DAR?